plugins/woocommerce-blocks/assets/js/blocks/mini-cart/frontend.ts

```diff
@@ -158,6 +161,16 @@ window.addEventListener( 'load', () => {
 			loadContents();
 		};
 
+		// Load the scripts if a device is touch-enabled. We don't get the mouseover or focus events on touch devices,
+		// so the event listeners below won't work.
+		if (
+			'ontouchstart' in window ||
+			navigator.maxTouchPoints > 0 ||
+			window.matchMedia( '(pointer:coarse)' ).matches
+		) {
+			loadScripts();
+		}
+
 		miniCartButton.addEventListener( 'mouseover', loadScripts );
 		miniCartButton.addEventListener( 'focus', loadScripts );
```

Comment:
Should we prevent adding listeners if it is touch enabled? I know the method bails early, still we don't need them triggering

Reply:
Good idea, not sure how I missed that.